packages/bp/src/common/type-coersion.test.ts

```diff
@@ -0,0 +1,31 @@
+import { isNil } from './type-coersion'
+
+describe('type-coersion', () => {
```

Comment:
this is useless test, use and trust lodash

Reply:
Done